tests/playwright/src/specs/docker-compatibility-smoke.spec.ts

```diff
@@ -0,0 +1,141 @@
+
+test.describe.serial('Verify docker compatibility feature', { tag: '@smoke' }, () => {
+  test('Enable the docker compatibility experimental feature', async ({ navigationBar, page }) => {
+    //Verify Settings
+    await navigationBar.openSettings();
+    const settingsBar = new SettingsBar(page);
+
+    const DCLink = settingsBar.getPreferencesLinkLocator(settingsDCLabel);
+    await playExpect(DCLink).not.toBeVisible();
+
+    //Enable the feature
+    await setDockerCompatibilityFeature(true, page);
+
+    await playExpect(DCLink).toBeVisible();
+  });
+  test('Verify Docker Compatibility page', async ({ page }) => {
+    const settingsBar = new SettingsBar(page);
+    const dockerCompatibilityPage = await settingsBar.openTabPage(DockerCompatibilityPage);
+    await playExpect(dockerCompatibilityPage.heading).toBeVisible();
+    playExpect(await dockerCompatibilityPage.socketIsReachable()).toBeTruthy();
+    await playExpect(dockerCompatibilityPage.serverInformationBox).toBeVisible();
+  });
+  test('Verify socket reachability is responding to podman machine status', async ({ page }) => {
+    const settingsBar = new SettingsBar(page);
+    await settingsBar.openTabPage(ResourcesPage);
+    const podmanMachineDetails = new PodmanMachineDetails(page, defaultMachine);
+    await playExpect(podmanMachineDetails.podmanMachineStopButton).toBeEnabled();
+    await podmanMachineDetails.podmanMachineStopButton.click();
+    await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('OFF', { timeout: 50_000 });
+
+    const dockerCompatibilityPage = await settingsBar.openTabPage(DockerCompatibilityPage);
+    playExpect(await dockerCompatibilityPage.socketIsReachable()).toBeFalsy();
+    await playExpect(dockerCompatibilityPage.serverInformationBox).not.toBeVisible();
+
+    await settingsBar.openTabPage(ResourcesPage);
+    await playExpect(podmanMachineDetails.podmanMachineStartButton).toBeEnabled();
+    await podmanMachineDetails.podmanMachineStartButton.click();
+    await playExpect(podmanMachineDetails.podmanMachineStatus).toHaveText('RUNNING', { timeout: 50_000 });
+
+    await settingsBar.openTabPage(DockerCompatibilityPage);
+    playExpect(await dockerCompatibilityPage.socketIsReachable()).toBeTruthy();
+    await playExpect(dockerCompatibilityPage.serverInformationBox).toBeVisible();
+  });
+  test('Disable docker compatibility', async ({ page }) => {
+    await setDockerCompatibilityFeature(false, page);
+
+    const settingsBar = new SettingsBar(page);
+    const DCLink = settingsBar.getPreferencesLinkLocator(settingsDCLabel);
+    await playExpect(DCLink).not.toBeVisible();
+  });
+});
+
+async function setDockerCompatibilityFeature(enable: boolean, page: Page): Promise<void> {
+  //Open the preferences bar and verify DC preferences page
+  const settingsBar = new SettingsBar(page);
+  await settingsBar.expandPreferencesTab();
+
+  const DCPreferencesLink = settingsBar.getPreferencesLinkLocator(settingsDCLabel);
```

Comment:
looks like this is working, right? You can get a different locator from settings/preferences even though they have the same aria-label? 
If not, we can check for the elements attribute using `locator.evaluate` and getting attribute (`href`) value where we could distinguish among links in settings and preferences.